grass7/imagery/i.sentinel/i.sentinel.import/i.sentinel.import.py

```diff
@@ -320,11 +320,12 @@ def create_register_file(self, filename):
                     ts=timestamp_str
                 ))
                 if has_band_ref:
+                    band_ref = re.match(
+                        r'.*_B([0-1][0-9]).*', map_name
```

Comment:
```suggestion
                        r'.*_B([0-1][0-9]).*', map_name
```
does this also consider B08A??

Reply:
@veroandreo Good catch, fixed in  a600ba5